src/open_inwoner/scss/components/Card/Card.scss

```diff
@@ -16,6 +16,33 @@
   position: relative;
   text-decoration: none;
 
+  &--flex {
+    display: flex;
+  }
+
+  &--spaced {
+    margin-bottom: var(--spacing-large);
+  }
+
+  @media (min-width: 768px) {
+    &--compact {
+      width: 50%;
+    }
+  }
+
+  .left {
+    float: left;
+  }
+  .right {
+    float: right;
+  }
+  .radio {
```

Comment:
Alignment seems a bit off, I'm not sure if we want to fix this in this PR or if Jiro will take a look afterwards

![image](https://github.com/maykinmedia/open-inwoner/assets/29249171/31db8a3b-72bd-4914-af6c-cd638066c8bc)

Reply:
Discussed with Steven, please re-use the login-button component instead of modifying the cards. Jiro can indeed make this more in line with the design later and I don't want to modify components which are used allover the place

Reply:
Modified it and used the existing templatetags, which removes the need for this extra CSS

![image](https://github.com/maykinmedia/open-inwoner/assets/29249171/b04a0158-bd24-4bba-ba6b-8d1eada68480)